cmd/doc/main.go

```diff
@@ -53,23 +57,35 @@ type SchemaPlusParent struct {
 	Schema map[string]apiextensions.JSONSchemaProps
 }
 
-var docTemplate = template.Must(template.New("doc.html").Funcs(
-	template.FuncMap{
-		"plusParent": func(p string, s map[string]apiextensions.JSONSchemaProps) *SchemaPlusParent {
-			return &SchemaPlusParent{
-				Parent: p,
-				Schema: s,
-			}
+var page = render.New(render.Options{
+	Extensions:    []string{".html"},
+	Directory:     "template",
+	Layout:        "layout",
+	IsDevelopment: os.Getenv(envAnalytics) == "true",
```

Comment:
should this be checking the new `envDevelopment` instead?

Reply:
Derp. Fixed